pages.es/common/gocr.md

```diff
@@ -0,0 +1,17 @@
+# gocr
+
+> Herramienta de reconocimiento óptico de caracteres.
+> Reconoce caracteres utilizando su motor y solicita al usuario patrones desconocidos para almacenarlos en una base de datos.
+> Más información: <https://manned.org/gocr.1>.
+
+- Reconoce caracteres en una [i]magen y los escribe en un archiv[o]. Coloca la base de datos en una carpeta existente para que no se omita su uso. [m]odo 130 significa crear, usar y extender la base de datos:
```

Comment:
```suggestion
- Reconoce caracteres en una [i]magen y los escribe en un archiv[o]. Coloca la base de datos en una carpeta existente para que no se omita su uso. [m]odo 130 significa crear, usar y extender la base de datos:
```

Now that I think about it, maybe the tense for the sentence `Coloca la base de datos en un carpeta existente` reads like an order rather than something the user should take into account.

What do you think @kant @kbdharun?